docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>just a nitpick:</p><p>[dali_types.UINT8 for i in range(3)] -&gt; [dali_types.UINT8 for _ in range(3)] or</p><p>[dali_types.UINT8 for i in range(3)] -&gt; [dali_types.UINT8] * 3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>